tests/comparisons/Seeds/testWithDataAndLimit.php

```diff
@@ -0,0 +1,39 @@
+<?php
+use Migrations\AbstractSeed;
+
+/**
+ * Events seed.
+ */
+class EventsSeed extends AbstractSeed
+{
+    /**
+     * Run Method.
+     *
+     * Write your database seeder using this method.
+     *
+     * More information on writing seeders is available here:
+     * http://docs.phinx.org/en/latest/seeding.html
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $data = [
+            [
+                'id' => '1',
+                'title' => 'Lorem ipsum dolor sit amet',
+                'description' => 'Lorem ipsum dolor sit amet, aliquet feugiat.',
+                'published' => 'Y',
+            ],
+            [
+                'id' => '2',
+                'title' => 'Second event',
+                'description' => 'Second event description.',
+                'published' => 'Y',
+            ],
```

Comment:
@lorenzo I like them too, but cake core omits them as far as I know.
I'd be great to have an official stance on this.